firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandler.java

```diff
@@ -286,16 +297,17 @@ private String createThrottledMessage(long throttledDurationInMillis) {
    *     error connecting to the server.
    */
   @WorkerThread
-  private FetchResponse fetchFromBackend(InstanceIdResult instanceId, Date currentTime)
+  private FetchResponse fetchFromBackend(
+      String installationId, String installationToken, Date currentTime)
       throws FirebaseRemoteConfigException {
     try {
       HttpURLConnection urlConnection = frcBackendApiClient.createHttpURLConnection();
 
       FetchResponse response =
           frcBackendApiClient.fetch(
               urlConnection,
-              instanceId.getId(),
-              instanceId.getToken(),
+              installationId,
+              installationToken,
```

Comment:
instanceId.getToken is FCM token and FIS getToken is FIS Auth Token.
Those are diff tokens. Just want to be sure this change is intended.

Reply:
Yep, thanks for checking. The RC backend's already been updated to accept either.

Reply:
Sounds good. Thanks for confirming :)

Reply:
Potentially add the term "auth" to the variable name?